app/src/main/java/co/develhope/chooseyouowncocktail_g2/DrinksDB.kt

```diff
@@ -0,0 +1,126 @@
+package co.develhope.chooseyouowncocktail_g2
+
+import DrinkResult
+import DrinkResultModel
+import co.develhope.chooseyouowncocktail_g2.mapper.DrinkMapper
+import co.develhope.chooseyouowncocktail_g2.network.DrinkDao
+import co.develhope.chooseyouowncocktail_g2.network.RestClient
+import java.util.*
+
+enum class Alchool {
+    Alcoholic, Non_Alcoholic
+}
+
+
+object DrinksDB {
```

Comment:
Perchè lo chiami DrinkDB? Io che atterro su questo mi aspetto un database ma in realtà ho delle chiamate di rete